src/bin/www.js

```diff
@@ -35,25 +38,33 @@ const server = http.createServer(app);
 // next code block goes here
 
 /**
- * Event listener for HTTP server "error" event.
+ *
+ * @param {obj} error - error
+ *
+ * @return {error} - handle specific listen errors with messages
  */
-const onError = error => {
+const onError = (error) => {
   if (error.syscall !== 'listen') {
     throw error;
   }
   const bind = typeof port === 'string' ? `Pipe ${port}` : `Port ${port}`;
-  // handle specific listen errors with friendly messages
   switch (error.code) {
-  case 'EACCES': alert(`${bind} requires elevated privileges`);
-    process.exit(1);
-    break; case 'EADDRINUSE': alert(`${bind} is already in use`);
-    process.exit(1);
-    break; default: throw error;
+    // eslint-disable-next-line no-alert
+    case 'EACCES': alert(`${bind} requires elevated privileges`);
```

Comment:
Why are we using `alert`? and not `console.log`. By the way I thought `alert` are dialogue boxes that displays in the browser(client), does it work on the server?